packages/lib/src/components/Ach/components/AchInput/types.ts

```diff
@@ -58,4 +58,11 @@ export interface ACHInputProps {
     styles?: StylesObject;
     type?: string;
     forceCompat?: boolean;
+    setComponentRef?: (ref) => void;
+}
+
+// An interface for the members exposed by a component to its parent UIElement
+export interface AchRef {
+    showValidation?: (who) => void;
```

Comment:
Regarding this `who` object - do we ever pass something to the `showValidation` function? Perhaps we don't need it

Reply:
Yeah, I think you're right